dev-docs/adunit-reference.md

```diff
@@ -0,0 +1,342 @@
+|----------------+----------+---------+------------------------------------------------------------------------------------------------------|
+| `min_width`    | Optional | Integer | The minimum width required for an image to serve (in pixels).                                        |
+| `ratio_height` | Required | Integer | This, combined with `ratio_width`, determines the required aspect ratio for an image that can serve. |
+| `ratio_width`  | Required | Integer | See above.                                                                                           |
+
+<a name="native-title" />
+
+##### `adUnit.mediaTypes.native.title`
+
+{: .table .table-bordered .table-striped }
+| Name       | Scope    | Type    | Description                                          |
+|------------+----------+---------+------------------------------------------------------|
+| `required` | Required | Boolean | Whether a title asset is required on this native ad. |
+| `len`      | Optional | Integer | Length of title text, in characters.                 |
+
+<a name="native-sponsoredBy" />
+
+##### `adUnit.mediaTypes.native.sponsoredBy`
+
+{: .table .table-bordered .table-striped }
+| Name       | Scope    | Type    | Description                                               |
+|------------+----------+---------+-----------------------------------------------------------|
+| `required` | Required | Boolean | Whether a brand name asset is required on this native ad. |
+
+<a name="native-clickUrl" />
+
+##### `adUnit.mediaTypes.native.clickUrl`
+
+{: .table .table-bordered .table-striped }
+| Name       | Scope    | Type    | Description                                              |
+|------------+----------+---------+----------------------------------------------------------|
+| `required` | Required | Boolean | Whether a click URL asset is required on this native ad. |
+
+##### `adUnit.mediaTypes.native.body`
+
+{: .table .table-bordered .table-striped }
+| Name       | Scope    | Type    | Description                                       |
+|------------+----------+---------+---------------------------------------------------|
+| `required` | Required | Boolean | Whether body text is required for this native ad. |
+
+<a name="native-icon" />
+
+##### `adUnit.mediaTypes.native.icon`
+
+{: .table .table-bordered .table-striped }
+| Name            | Scope    | Type                                  | Description                                                                                                                       |
+|-----------------+----------+---------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------|
+| `required`      | Requird  | Boolean                               | Whether an icon asset is required on this ad.                                                                                     |
```

Comment:
Typo on "Requird" (but should probably be "Optional" anyway).

Reply:
Updated all mentions of the required property to note they are optional in dbc79b2